portability-gateway/src/main/java/org/dataportabilityproject/gateway/ApiMain.java

```diff
@@ -78,29 +88,37 @@ public void initializeHttps(
 
     ExtensionContext extensionContext = new ApiExtensionContext(typeManager, configuration);
 
+    // Services that need to be shared between extensions or load types in the typemanager get
+    // initialized first.
     ServiceLoader.load(ServiceExtension.class)
         .iterator()
         .forEachRemaining(serviceExtension -> serviceExtension.initialize(extensionContext));
 
     CloudExtension cloudExtension = getCloudExtension();
     cloudExtension.initialize(extensionContext);
 
-    // TODO: Support other server implementations, e.g. Jetty, Tomcat
-    // TODO: Don't hardcode list of services
-    Set<String> services = ImmutableSet.of("microsoft");
+    // TODO: Load up only "enabled" services
+    List<AuthServiceExtension> authServiceExtensions = new ArrayList<>();
+    ServiceLoader.load(AuthServiceExtension.class)
+        .iterator()
+        .forEachRemaining(
+            (serviceExtension) -> {
```

Comment:
can you rename this local var authServiceExtension? I confused it with serviceExtension above

Reply:
Done